packages/web-app-files/src/components/SideBar/Shares/SpaceMembers.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div id="oc-files-sharing-sidebar" class="oc-position-relative">
+  <div id="oc-files-sharing-sidebar" class="oc-position-relative" data-custom-key-bindings="true">
```

Comment:
`data-custom-key-bindings="true"` can be removed because `SideBar.vue` already has it (parent container)